namespaces/exec.go

```diff
@@ -255,7 +255,7 @@ func AddUidGidMappings(sys *syscall.SysProcAttr, container *libcontainer.Config)
 //
 // console: the /dev/console to setup inside the container
 // init: the program executed inside the namespaces
-// root: the path to the container json file and information
+// dataPath: the path saving container's states
```

Comment:
Could you change it to something like
dataPath: the path to the directory under which the container's state file is stored